Docker/validate.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+"""Validation script for scRNA-seq signal correction.
+Predictions file must be a zipped tarball archive of imputed count files:
+(*.tar.gz).
+Each imputed count file must have a correct file format:
+(pac_expId_ds_prop_imputed.csv).
+"""
+
+import argparse
+import json
+import pandas as pd
+from zipfile import ZipFile
+
+
+def get_args():
+    """Set up command-line interface and get arguments."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-r", "--results", required=True,
+                        help="validation results")
+    parser.add_argument("-e", "--entity_type", required=True,
+                        help="synapse entity type downloaded")
+    parser.add_argument("-s", "--submission_file", help="Submission File")
+    parser.add_argument("-g", "--goldstandard", required=True,
+                        help="Goldstandard for scoring")
+    return parser.parse_args()
+
+
+def get_dim_names(df):
+    """
+    Get all row names and column names of df
+    """
+    out = {"rownames": df.index.tolist(),
+           "colnames": df.columns.values.tolist()}
+    return out
+
+
+def main():
+    """Main function."""
+    args = get_args()
+
+    invalid_reasons = []
+    gs_file_status = True
+    prediction_file_status = True
+    exp_ids = ["2400", "2401", "7200", "7201"]
+    ds_props = ["0_125"]
+
+    # validate goldstandard file
+    if args.goldstandard is None:
```

Comment:
`goldstandard` is always going to be provided by the CWL params, right?  Is this check for local testing?